### ISSUE TITLE
DoubleRenderError when inputing an invalid token

### ISSUE THREAD

**Issue by rdimartino** [exop-group/doorkeeper-device_authorization_grant#12]
On version 1.0.2, entering an invalid token on `/oauth/device` leads to an error

```
AbstractController::DoubleRenderError in Doorkeeper::DeviceAuthorizationGrant::DeviceAuthorizationsController#authorize 
```

I can recreate this in the example repo https://github.com/exop-group/doorkeeper-device-flow-example by changing the `backend/Gemfile` version of this gem

```
gem 'doorkeeper-device_authorization_grant', '1.0.2'
```

1.0.0 and 1.0.1 both work

**Comment by irminsul** [comment]
Thanks for sharing this!

I wonder why we did not catch this earlier. People enter invalid codes in our own IAM portal all the time and this type of error never popped up in the monitoring.

Most of our team is on leave over the Easter weekend, so unfortunately we won't be able to look into this until next Tuesday. Stay tuned!

**Comment by irminsul** [comment]
Fixed in v1.0.3
